Dockerfile

```diff
@@ -18,6 +18,7 @@ RUN apk add --update ca-certificates
 COPY --from=builder /app/backup /usr/bin/backup
 
 COPY ./entrypoint.sh /root/
+COPY ./00notifications.tmpl /etc/volume-backup.d/
```

Comment:
This is very nit-picky, but I think I'd prefer if the file in the repo was called `notifications.tmpl` only and we prepend the `00` when copying:
```
COPY ./notifications.tmpl /etc/volume-backup.d/00defaults.tmpl
```

Reply:
No problem!